windowsforms/FlexGridShowcaseDemo/cs/FlexGridShowcaseDemo.csproj

```diff
@@ -1,8 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
```

Comment:
Is this SDK not used anymore?

Reply:
It is no longer required, there was a compiler warning stating that it can safely be replaced.